systems/andes_user_guide.rst

```diff
@@ -1151,3 +1151,294 @@ The My OLCF site is provided to aid in the utilization and management of OLCF
 allocations. If you have any questions or have a request for additional data,
 please contact the OLCF User Assistance Center.
 
+--------------
+
+.. _visualization-tools:
+
+Visualization tools
+====================
+
+Remote Visualization using VNC (non-GPU)
+----------------------------------------
+
+In addition to the instructions below, `Benjamin
+Hernandez <https://www.olcf.ornl.gov/directory/staff-member/benjamin-hernandez/>`__ of the `OLCF
+Advanced Technologies
+Section <https://www.olcf.ornl.gov/about-olcf/staff-sections/advanced-technologies/>`__
+presented a related talk, `GPU Rendering in Rhea and
+Titan <https://www.olcf.ornl.gov/wp-content/uploads/2016/01/GPURenderingRheaTitan-1.pdf>`__,
+during the 2016 OLCF User Meeting.
+
+Step 1 (local system)
+^^^^^^^^^^^^^^^^^^^^^
+
+Install a vncviewer (turbovnc, tigervnc, etc.) on your local machine.  When
+running vncviewer for the first time, it will ask to set a password for this and
+future vnc sessions.
+
+Step 2 (terminal 1)
+^^^^^^^^^^^^^^^^^^^
+
+From an Andes connection launch a batch job and execute the below matlab-vnc.sh
+script to start the vncserver and run matlab within:
+
+#. localsytem: ``ssh -X username@andes.olcf.ornl.gov``
+#. andes: ``salloc -A abc123 -N 1 -t 1:00:00 --x11=batch``
+#. andes: ``./matlab-vnc.sh``
+
+.. code::
+
+    $ ./matlab-vnc.sh
+
+    Starting vncserver
+
+    Desktop 'TurboVNC: andes79.olcf.ornl.gov:1 (userA)' started on display andes79.olcf.ornl.gov:1
+
+    Starting applications specified in /ccs/home/userA/.vnc/xstartup.turbovnc
+    Log file is /ccs/home/userA/.vnc/andes79.olcf.ornl.gov:1.log
+
+    **************************************************************************
+    Instructions
+
+    In a new terminal, open a tunneling connection with andes79.olcf.ornl.gov and port 5901
+    example:
+         username@andes.olcf.ornl.gov -L 5901:andes79.olcf.ornl.gov:5901
```

Comment:
Do we now need two hops?
ssh -L 5901:localhost:5901 <username>@andes.olcf.ornl.gov
ssh -L 5901:localhost:5901 andes79

Reply:
Yes I have to use 2 hops to make it work